Lib/test/test__opcode.py

```diff
@@ -2,6 +2,7 @@
 from test.support.import_helper import import_module
 import unittest
 import opcode
+import _opcode_metadata
```

Comment:
Would it make sense for `opcode` to import from `_opcode_metadata`, so that the API doesn't change? (although it is only the private API)

Reply:
Could do.